packages/interface/src/components/complementary-area-toggle/index.js

```diff
@@ -35,6 +35,13 @@ function ComplementaryAreaToggle( {
 		<ComponentToUse
 			icon={ selectedIcon && isSelected ? selectedIcon : icon }
 			aria-controls={ identifier.replace( '/', ':' ) }
+			// Make sure aria-checked matches spec https://www.w3.org/TR/wai-aria-1.1/#aria-checked
+			aria-checked={
+				props.role === 'menuitemcheckbox' ||
+				props.role === 'menuitemradio'
+					? isSelected
+					: undefined
+			}
```

Comment:
This is the same approach used in

https://github.com/WordPress/gutenberg/blob/8c5a0b6a4f5e0802d0c84d1f8e6e4934c7a449b3/packages/components/src/menu-item/index.tsx#L60-L65

Reply:
I think the component rendered by plugins will always be `MenuItem`. See:

https://github.com/WordPress/gutenberg/blob/69efbca3ba4522b9b11c56a5e517d123fe9a9071/packages/interface/src/components/complementary-area-more-menu-item/index.js#L13-L50

Reply:
Updated. Thanks.

Reply:
> I think the component rendered by plugins will always be `MenuItem`

That's true, but `<ComplementaryAreaToggle />` is a generic component that technically can be rendered `as` anything, and therefore I don't think that checking for more roles can do any harm in this case